pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -190,120 +121,133 @@ class GatewayDataForm extends React.Component {
-          description={m.updateChannelDescription}
-          placeholder={sharedMessages.stable}
-          name="update_channel"
-          component={Input}
-          autoComplete="on"
-        />
+        <Collapse
+          title={m.basicTitle}
+          description={m.basicDescription}
+          disabled={false}
+          initialCollapsed={false}
+        >
+          <Message component="h4" content={sharedMessages.generalSettings} />
+          {!update && <OwnersSelect name="owner_id" required autoFocus />}
+          <Form.Field
+            title={sharedMessages.gatewayID}
+            name="ids.gateway_id"
+            placeholder={m.gatewayIdPlaceholder}
+            required
+            disabled={update}
+            component={Input}
+          />
+          <Form.Field
+            title={sharedMessages.gatewayEUI}
+            name="ids.eui"
+            type="byte"
+            min={8}
+            max={8}
+            placeholder={sharedMessages.gatewayEUI}
+            component={Input}
+          />
+          <Form.Field
+            title={sharedMessages.gatewayName}
+            placeholder={m.gatewayNamePlaceholder}
+            name="name"
+            component={Input}
+          />
+          <Form.Field
+            title={sharedMessages.gatewayDescription}
+            description={m.gatewayDescDescription}
+            placeholder={m.gatewayDescPlaceholder}
+            name="description"
+            type="textarea"
+            component={Input}
+          />
+          <Form.Field
+            title={sharedMessages.gatewayServerAddress}
+            description={m.gsServerAddressDescription}
+            placeholder={sharedMessages.addressPlaceholder}
+            name="gateway_server_address"
+            component={Input}
+          />
+          <Form.Field
+            title={sharedMessages.gatewayStatus}
+            name="status_public"
+            component={Checkbox}
+            label={sharedMessages.public}
+            description={m.statusDescription}
+          />
+          <Form.Field
+            name="attributes"
+            title={sharedMessages.attributes}
+            keyPlaceholder={sharedMessages.key}
+            valuePlaceholder={sharedMessages.value}
+            addMessage={sharedMessages.addAttributes}
+            component={KeyValueMap}
+            description={sharedMessages.attributeDescription}
+          />
+          <Message component="h4" content={sharedMessages.gatewayUpdateOptions} />
+          <Form.Field
+            title={sharedMessages.automaticUpdates}
+            name="auto_update"
+            component={Checkbox}
+            description={m.autoUpdateDescription}
+          />
+          <Form.Field
+            title={sharedMessages.channel}
+            description={m.updateChannelDescription}
+            placeholder={sharedMessages.stable}
+            name="update_channel"
+            component={Input}
+            autoComplete="on"
+          />
```

Comment:
I feel like these could either be put into a separate section or be merged with `LoRaWAN Options`. @benolayinka do you have an idea for a better separation?

Reply:
I don't think it matters. They aren't really LoRaWAN options. What kind of updates are these, though? And the "stable" prompt can be improved

<img width="685" alt="Screenshot 2020-10-20 at 16 46 13" src="https://user-images.githubusercontent.com/6963436/96603135-062f1080-12f4-11eb-9b1c-62e078120127.png">